include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -48,84 +48,77 @@ namespace __internal
 //-----------------------------------------------------------------------------
 
 #if _ONEDPL_DEBUG_SYCL
-template <typename _ExecutionPolicy>
-::std::string
-__device_info(const _ExecutionPolicy& __policy)
+inline std::string
+__device_info(const sycl::queue& __q)
 {
-    return __policy.queue().get_device().template get_info<sycl::info::device::name>();
+    return __q.get_device().template get_info<sycl::info::device::name>();
 }
 #endif
 
-template <typename _ExecutionPolicy>
-std::size_t
-__max_work_group_size(const _ExecutionPolicy& __policy, std::size_t __wg_size_limit = 8192)
+inline std::size_t
+__max_work_group_size(const sycl::queue& __q, std::size_t __wg_size_limit = 8192)
 {
-    std::size_t __wg_size = __policy.queue().get_device().template get_info<sycl::info::device::max_work_group_size>();
+    std::size_t __wg_size = __q.get_device().template get_info<sycl::info::device::max_work_group_size>();
```

Comment:
Same comment with regards to the `template` keyword.

Reply:
https://github.com/uxlfoundation/oneDPL/issues/2173